### PR TITLE
Correct docker build warnings

### PR DESCRIPTION
**Pull request by steveoh**
image

### CODE REVIEW

File: cloudrun/Dockerfile [hunk 1]
```diff
@@ -22,18 +22,16 @@ ENV CLOUDSDK_CONFIG="/gcp/config/application_default_credentials.json"
...
+FROM base AS prod
 
 COPY . .
 
 RUN pip install .
 RUN pip install arcgis==2.*
 RUN pip install gunicorn
 
-CMD exec gunicorn --bind :$PORT --timeout 0 download.main:app
+CMD ["exec", "gunicorn", "--bind", ":$PORT", "--timeout", "0", "download.main:app"]
```
Comment (steveoh) [inline cloudrun/Dockerfile:37]: **question**: the exec piece may be unnecessary. I'm not as confident about this update.

Comment (stdavis) [reply cloudrun/Dockerfile]: It sounds like [both forms are supported](https://docs.docker.com/reference/dockerfile/#shell-and-exec-form). The shell form seems easier to read and maintain. But I don't have any major problem with it.

Comment (steveoh) [reply cloudrun/Dockerfile]: Yeah i'm not sure what issues it can solve but the warning and fix seemed easy enough that I thought we should include it.

**Review by stdavis — APPROVED** [review]
I've tested this locally.

**Pull request merged**